user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -68,16 +79,17 @@ The elements will be created with the following specifications:
   - **IP address**: 127.0.0.1.
   - **Alarm template**: public.
   - **Trend template**: private.
-  - **Get community string**: The defined community string.
-  - **Entity import directory**: `\\FE DMA Name\DataMiner EPM\DOCSIS`.
+  - **Get community string Collector**: The defined get community string for the collector.
+  - **Set community string Collector**: The defined set community string for the collector.
+  - **Entity import directory**: The same directory as is set in the back end of the host.
   - **Entity import directory type**: Remote.
   - **System username**: The system username.
   - **System password**: The system password.
 
```

Comment:
Is the system username/password, the one requested in previous steps? And if so, can we update the naming here:
   1. Enter the user for this device.
   1. Enter the password for this device.

Reply:
@EstebanSkyline, could you confirm if these are the same credentials as mentioned earlier?

Reply:
@MariekeGO Yes, they are the credentials mentioned earlier